### DISCUSSION TITLE
I want use micro frontend with Nuxt 3

### DISCUSSION THREAD

**Discussion by Chuong2203** [nuxt/nuxt#22338] [Questions]
### Describe the feature

I want use micro frontend with Nuxt 3 by webpack. can u tutorial for me?

### Additional information

- [X] Would you be willing to help implement this feature?
- [X] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by WailanTirajoh** [comment]
Module Federation by webpack or with multiple nuxt with proxy?

**Reply by Chuong2203** [reply]
Yes sir, i tried it a lots in nuxt.config.js but it not working. Can u help me,pls?
